ci/ima.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Assumes that the current environment is a mutable ostree-container
+# with ostree-ext-cli installed in /usr/bin.  
+# Runs IMA tests.
+set -xeuo pipefail
+
+if test '!' -x /usr/bin/evmctl; then
+    rpm-ostree install ima-evm-utils
```

Comment:
It's tempting to add support for IMA using the shared library.  Alternatively, I might argue to just add this FCOS by default.

Or, we could use a separate fixture for this.  Dunno.

Reply:
If you mean adding `ima-evm-utils` to the FCOS default builds and not just adding it here... I think probably the path of least resistance would be to add a separate fixture.